crates/types/src/config/common.rs

```diff
@@ -161,21 +162,23 @@ pub struct CommonOptions {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub storage_low_priority_bg_threads: Option<NonZeroUsize>,
 
-    /// # Total memory limit for rocksdb caches and memtables. This includes memory
-    /// for uncompressed block cache and all memtables by all open databases.
+    /// # Total memory limit for rocksdb caches and memtables.
     ///
-    /// The memory size used for rocksdb caches. Default is 4GB.
-    pub rocksdb_total_memory_limit: u64,
+    /// This includes memory for uncompressed block cache and all memtables by all open databases.
+    /// The memory size used for rocksdb caches.
+    #[serde_as(as = "ByteCount<false>")]
+    #[cfg_attr(feature = "schemars", schemars(with = "ByteCount<false>"))]
+    pub rocksdb_total_memory_size: u64,
```

Comment:
Related to my previous comment. If we want this field to be non zero, should we change it to `NonZeroU64`?